packages/contracts/src/addresses.ts

```diff
@@ -187,3 +191,146 @@ export const getCheckBalanceLowAddress = () => {
     solidityPackedKeccak256(['bytes'], [CheckBalanceLowArtifact.bytecode])
   )
 }
+
+/**
+ * Sorts an array of hex strings in ascending order and returns the sorted array. Does not mutate
+ * the original array.
+ *
+ * @param arr The array of hex strings to sort.
+ * @returns A new sorted array.
+ */
+export const sortHexStrings = (arr: Array<string>): Array<string> => {
+  // Create a copy of the array
+  const arrCopy = [...arr]
+
+  // Sort the copied array
+  return arrCopy.sort((a, b) => {
+    const aBigInt = BigInt(a)
+    const bBigInt = BigInt(b)
+    return aBigInt < bBigInt ? -1 : aBigInt > bBigInt ? 1 : 0
+  })
+}
+
+export const getGnosisSafeInitializerData = (
```

Comment:
Since project creation occurs entirely via the website, we need a method for calculating the safe address in JavaScript to store in the DB. This function was originally in the test files, I pulled it into the contracts package and added some tests for it so we can use it in production.